rllib/evaluation/sampler.py

```diff
@@ -546,11 +613,13 @@ def new_episode():
             policies=policies,
             active_episodes=active_episodes,
             tf_sess=tf_sess,
-            _use_trajectory_view_api=_use_trajectory_view_api)
+            _use_trajectory_view_api=_use_trajectory_view_api,
+            _sample_collector=_sample_collector,
+        )
         perf_stats.inference_time += time.time() - t2
 
         # Process results and update episode state.
-        t3 = time.time()
+        t = time.time()
```

Comment:
For clarity, prefer to assign new variables instead of mutating existing ones (please revert to the t1, t2, t3 etc).

Reply:
done